<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionSettingsDialog</class>
 <widget class="QDialog" name="ConnectionSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QGroupBox" name="p_groupBox_BaudRate">
     <property name="title">
      <string>baudRate</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="7" column="1">
       <widget class="QRadioButton" name="p_radioButton_4800">
        <property name="text">
         <string>4800</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QRadioButton" name="p_radioButton_230400">
        <property name="text">
         <string>230 400</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QRadioButton" name="p_radioButton_19200">
        <property name="text">
         <string>19 200</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="p_radioButton_1200">
        <property name="text">
         <string>1200</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QRadioButton" name="p_radioButton_2400">
        <property name="text">
         <string>2400</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QRadioButton" name="p_radioButton_76800">
        <property name="text">
         <string>76 800</string>
        </property>
       </widget>
      </item>
      <item row="17" column="1">
       <widget class="QRadioButton" name="p_radioButton_576000">
        <property name="text">
         <string>576 000</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QRadioButton" name="p_radioButton_28800">
        <property name="text">
         <string>28 800</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="p_radioButton_300">
        <property name="text">
         <string>300</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QRadioButton" name="p_radioButton_38400">
        <property name="text">
         <string>38 400</string>
        </property>
       </widget>
      </item>
      <item row="18" column="0">
       <widget class="QRadioButton" name="p_radioButton_921600">
        <property name="text">
         <string>921 600</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QRadioButton" name="p_radioButton_57600">
        <property name="text">
         <string>57 600</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QRadioButton" name="p_radioButton_115200">
        <property name="text">
         <string>115 200</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QRadioButton" name="p_radioButton_460800">
        <property name="text">
         <string>460 800</string>
        </property>
       </widget>
      </item>
      <item row="23" column="0">
       <widget class="QRadioButton" name="p_radioButton_Custom">
        <property name="text">
         <string>Custom</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QRadioButton" name="p_radioButton_9600">
        <property name="text">
         <string>9600</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QRadioButton" name="p_radioButton_1800">
        <property name="text">
         <string>1800</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="p_radioButton_600">
        <property name="text">
         <string>600</string>
        </property>
       </widget>
      </item>
      <item row="23" column="1">
       <widget class="QLineEdit" name="p_lineEdit_CustomBaudRate">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="24" column="0" colspan="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="p_groupBox_Other">
     <property name="title">
      <string>Other</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Direction</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Parity</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>FlowControl</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>DataBits</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="p_comboBox_StopBits">
        <item>
         <property name="text">
          <string>OneStop</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OneAndHalfStop</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TwoStop</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="p_comboBox_Direction">
        <item>
         <property name="text">
          <string>Input</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Output</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AllDirections</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="p_comboBox_Parity">
        <item>
         <property name="text">
          <string>NoParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EvenParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OddParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SpaceParity</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MarkParity</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>StopBits</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="p_comboBox_FlowControl">
        <item>
         <property name="text">
          <string>NoFlowControl</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HardwareControl</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SoftwareControl</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="p_comboBox_DataBits">
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="p_groupBox_PinoutSignal">
     <property name="title">
      <string>PinoutSignal</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="p_checkBox_NoSignal">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>NoSignal</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="p_checkBox_DTR">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>(DTR) DataTerminalReadySignal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="p_checkBox_DCD">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>(DCD) DataCarrierDetectSignal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="p_checkBox_DSR">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>(DSR) DataSetReadySignal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="p_checkBox_RNG">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>(RNG) RingIndicatorSignal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="p_checkBox_RTS">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>(RTS) RequestToSendSignal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="p_checkBox_CTS">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>(CTS) ClearToSendSignal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="p_checkBox_STD">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>(STD) SecondaryTransmittedDataSignal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="p_checkBox_SRD">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>(SRD) SecondaryReceivedDataSignal</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="p_lineEdit_Device">
     <property name="text">
      <string>ttyUSB0</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Device:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConnectionSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConnectionSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>p_radioButton_Custom</sender>
   <signal>toggled(bool)</signal>
   <receiver>p_lineEdit_CustomBaudRate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>519</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
